server/core/src/config.rs

```diff
@@ -179,41 +225,64 @@ pub struct Configuration {
     pub role: ServerRole,
     pub output_mode: ConsoleOutputMode,
     pub log_level: LogLevel,
+
+    /// Replication settings.
+    pub repl_config: Option<ReplicationConfiguration>,
+    /// This allows interally setting some unsafe options for replication.
+    pub integration_repl_config: Option<Box<IntegrationReplConfig>>,
```

Comment:
We probably should be gating the existence of this behind `#[cfg(any(debug_assertions,test))]` so it's not triggerable in prod...

Reply:
That's going to be a more fun change I expect .... 🤔 but it's possible.